### ISSUE TITLE
Unable to build from master

### ISSUE THREAD

**Issue by therealjohn** [WildernessLabs/NetMF_MonoDevelop_VS_Mac_Addin#15]
Cloning master and trying to follow the directions in the README results in build errors for missing files in the `/Debug` directory. 

For example, it's complaining about missing all these files: https://github.com/WildernessLabs/NetMF_MonoDevelop_VS_Mac_Addin/tree/8f8b7fbb7ac7154f96a5c4d3abcc87133540223c/VS%20Addin/MonoDevelop.MicroFramework/Debug

**Comment by JamestsaiTW** [comment]
Yes, VS Addin solution did miss /Debug directory. My workaround is clone previous commit to get /Debug directory's file, and copy them to the latest commit.
